src/components/Restoration/YourCompanySummary.vue

```diff
@@ -100,50 +100,49 @@
 </template>
 
 <script lang="ts">
-import { ApprovalTypes } from '@bcrs-shared-components/enums'
-import { mapState } from 'pinia'
-import { OfficeAddresses, NameTranslation } from '@/components/common/'
+import { Component, Vue } from 'vue-property-decorator'
+import { Getter } from 'pinia-class'
 import { useStore } from '@/store/store'
+import { ApprovalTypes, RelationshipTypes } from '@bcrs-shared-components/enums'
+import { OfficeAddresses, NameTranslation } from '@/components/common/'
+import { NameRequestIF } from '@bcrs-shared-components/interfaces'
+import { CourtOrderIF } from '@/interfaces/alteration-interfaces'
+import { RestorationStateIF } from '@/interfaces'
 
-export default {
+@Component({
   components: {
     OfficeAddresses,
     NameTranslation
-  },
-  data () {
-    return {
-      ApprovalTypes
-    }
-  },
-  computed: {
-    ...mapState(useStore, [
-      'getCourtOrderNumberText',
-      'getBusinessNumber',
-      'getFormattedExpiryText',
-      'getNameRequestLegalName',
-      'getNameRequest',
-      'getRelationships',
-      'getRestoration',
-      'getIsRestorationTypeCourtOrder',
-      'getStateFilingRestoration',
-      'hasBusinessNameChanged',
-      'haveNameTranslationsChanged',
-      'isLimitedRestorationExtension',
-      'isLimitedRestorationToFull'
-    ]),
-    getStateFilingApprovalType () {
-      return this.getStateFilingRestoration?.approvalType
-    },
-    getCourtOrder () {
-      return this.getRestoration.courtOrder
```

Comment:
For some reason, VS Code / Volar / eslint was happy with this but the build process wasn't happy with the type of "courtOrder"... so I refactored this component to Class API. All good now.